.gitignore

```diff
@@ -9,3 +9,4 @@ htmlcov
 .pytest_cache
 cclib.egg-info
 timer.dat
+.DS_Store
```

Comment:
What is this from?

Reply:
It's a directory metadata file created by Mac: https://en.wikipedia.org/wiki/.DS_Store. Created if you ever open up a directory (e.g. `data`) in Finder.